src/sdk-reference/js/6/getting-started/react/snippets/subscribe.js

```diff
@@ -0,0 +1,13 @@
+/* eslint no-invalid-this: 0 */
+this.props = {
+  setMessages: () => {}
+};
+
+/* snippet:start */
+kuzzle.realtime.subscribe('chat', 'messages', {}, notif => {
+  if (notif.type === 'document' && notif.action === 'create') {
+    const { _source: message } = notif.result;
```

Comment:
Nitpicking: Use a guard and return if the condition does not match to avoid unnecessary indentation